diff --git a/html/parsing/html5lib/html-tokenizer/unicodeCharsProblematic.test b/html/parsing/html5lib/html-tokenizer/unicodeCharsProblematic.test
index 5987845..accd2ae 100644
--- a/html/parsing/html5lib/html-tokenizer/unicodeCharsProblematic.test
+++ b/html/parsing/html5lib/html-tokenizer/unicodeCharsProblematic.test
@@ -2,22 +2,22 @@
 {"description": "Invalid Unicode character U+DFFF",
 "doubleEscaped":true,
 "input": "\\uDFFF",
-"output":["ParseError", ["Character", "\\uFFFD"]]},
+"output":["ParseError", ["Character", "\\uDFFF"]]},
 
 {"description": "Invalid Unicode character U+D800",
 "doubleEscaped":true,
 "input": "\\uD800",
-"output":["ParseError", ["Character", "\\uFFFD"]]},
+"output":["ParseError", ["Character", "\\uD800"]]},
 
 {"description": "Invalid Unicode character U+DFFF with valid preceding character",
 "doubleEscaped":true,
 "input": "a\\uDFFF",
-"output":[["Character", "a"], "ParseError", ["Character", "\\uFFFD"]]},
+"output":["ParseError", ["Character", "a\\uDFFF"]]},
 
 {"description": "Invalid Unicode character U+D800 with valid following character",
 "doubleEscaped":true,
 "input": "\\uD800a",
-"output":["ParseError", ["Character", "\\uFFFDa"]]},
+"output":["ParseError", ["Character", "\\uD800a"]]},
 
 {"description":"CR followed by U+0000",
 "input":"\r\u0000",
diff --git a/html/parsing/html5lib/html-tokenizer/xmlViolation.test b/html/parsing/html5lib/html-tokenizer/xmlViolation.test
index 137d964..d108c4c 100644
--- a/html/parsing/html5lib/html-tokenizer/xmlViolation.test
+++ b/html/parsing/html5lib/html-tokenizer/xmlViolation.test
@@ -3,16 +3,16 @@
 {"description":"Non-XML character",
 "input":"a\uFFFFb",
 "ignoreErrorOrder":true,
-"output":["ParseError",["Character","a\uFFFDb"]]},
+"output":["ParseError",["Character","a\uFFFFb"]]},
 
 {"description":"Non-XML space",
 "input":"a\u000Cb",
 "ignoreErrorOrder":true,
-"output":[["Character","a b"]]},
+"output":[["Character","a\u000Cb"]]},
 
 {"description":"Double hyphen in comment",
 "input":"<!-- foo -- bar -->",
-"output":["ParseError",["Comment"," foo - - bar "]]},
+"output":["ParseError",["Comment"," foo -- bar "]]},
 
 {"description":"FF between attributes",
 "input":"<a b=''\u000Cc=''>",
